debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbChangeRecordEmitter.java

```diff
@@ -13,6 +13,7 @@
 import org.bson.Document;
 
 import io.debezium.annotation.ThreadSafe;
+import io.debezium.connector.common.Partition;
```

Comment:
I know it is urelated to this PR but does not belog `Partition` to `spi` package as the `OffsetContext`?

Reply:
What does SPI stand for in this context? Happy to move it there (whether in this PR if more rework happens to be needed) or in a subsequent one.

Reply:
It stands for "service provider interface", and in this context here the `io.debezium.pipeline.spi` package is meant as the home for interfaces to be implemented by connectors based on the common CDC framework.

Reply:
That said, the package organization within the debezium-core artifact isn't very good and likely we'll need to do some clean-up eventually. It's one of the reasons why the connector framework isn't really a "supported" part of our API surface atm. (i.e. external connector using it are prone to break upon Debezium version upgrades).